builders/get-started/eth-compare/tx-fees.md

```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
 
 === "EIP-1559"
     ```
-    Transaction Fee =（Base Fee + Max Priority Fee Per Gas) * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    Gas Price = BaseFee + MaxPriorityFeePerGas < MaxFeePerGas ? 
+                BaseFee + MaxPriorityFeePerGas : 
+                MaxFeePerGas;
+    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
```

Comment:
```suggestion
    Gas Price = Base Fee + Max Priority Fee Per Gas < Max Fee Per Gas ? 
                Base Fee + Max Priority Fee Per Gas : 
                Max Fee Per Gas;
    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
```

all of the other variables have a space between the words so I would change these just to be consistent